frontend/ui/lib/widgets/trip_list_widget.dart

```diff
@@ -35,8 +36,33 @@ class TripListWidget extends StatelessWidget {
                 Navigator.pushNamed(
                     context, Router.createTripViewRoute(trip.id));
               },
+              trailing: Tooltip(
+                  message: 'Click here to delete trip.',
+                  child: IconButton(
+                    onPressed: () {
+                      _showDialog(context);
+                    },
+                    icon: Icon(Icons.delete),
+                    color: Colors.red,
+                  )),
             ))
         .toList();
     return ListView(children: listItems);
   }
+
+  _showDialog(BuildContext context) {
+    VoidCallback continueCallBack = () => {
+          Navigator.of(context).pop(),
+          //I don't think we have a delete trip method.
```

Comment:
Looks like we need to add one on the Dart side. We _do_ have a delete-trip servlet implementation (`TripServlet.doDelete`).